File: seed/csharp-sdk/pagination/src/SeedPagination/Users/Types/WithPage.cs
using System.Text.Json.Serialization;
using SeedPagination.Core;

namespace SeedPagination;

public record WithPage
{
    [JsonPropertyName("page")]
    public int? Page { get; set; }

    public override string ToString()
    {
        return JsonUtils.Serialize(this);
    }
}

File: seed/php-model/pagination/src/InlineUsers/InlineUsers/WithPage.php
<?php

namespace Seed\InlineUsers\InlineUsers;

use Seed\Core\Json\JsonSerializableType;
use Seed\Core\Json\JsonProperty;

class WithPage extends JsonSerializableType
{
    /**
     * @var ?int $page
     */
    #[JsonProperty('page')]
    public ?int $page;

    /**
     * @param array{
     *   page?: ?int,
     * } $values
     */
    public function __construct(
        array $values = [],
    )
    {
        $this->page = $values['page'] ?? null;
    }

    /**
     * @return string
     */
    public function __toString(): string {
        return $this->toJson();
    }
}
